src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques6 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    public boolean newMethod(String st1, String st2) {
+        if (st1 == st2) {
```

Comment:
Comparison using '==' for String objects. Use equals() method for String comparison.